source/localizable/testing/testing-components.md

```diff
@@ -126,7 +125,7 @@ test('should update title on button click', function(assert) {
   assert.equal(this.$('h2').text(), 'Hello World', 'initial text is hello world');
 
   //Click on the button
-  this.$('button').click();
+  document.getElementsByClassName('title-button')[0].click();
```

Comment:
@cibernox good points ;) Will update the description and include a mention for the run loop